src/index.js

```diff
@@ -0,0 +1,35 @@
+const task = [
+  {
+    index: [0],
+    description: 'Take car to Garage',
+    completed: false,
+  },
+  {
+    index: [1],
```

Comment:
- [ ] The index should be a number, not an array, kindly consider Implementing this by removing the `[ ]` around the numbers